contracts/TablelandTables.sol

```diff
@@ -166,6 +163,24 @@ contract TablelandTables is
         return _controllers[tableId];
     }
 
+    /**
+     * @dev See {ITablelandTables-lock}.
+     */
+    function lock(address caller, uint256 tableId)
+        external
+        override
+        whenNotPaused
+    {
+        if (
+            !_exists(tableId) ||
+            !(caller == _msgSenderERC721A() && caller == ownerOf(tableId))
+        ) {
+            revert Unauthorized();
+        }
+
+        transferFrom(caller, address(this), tableId);
```

Comment:
transferring to the contract's address, which some threads say is a good way to burn things. double check me though... contracts can't sign things so it's not like the contract owner is able to claim ownership right? happy to just use a burn address as well.

Reply:
Interesting, sounds like a good strategy but I'll read up on it

Reply:
I like it! I can't find any reason this wouldn't work, and it seems like more neutral address to use

Reply:
wondering though, should we hold off on this PR until we can support the flow @awmuncy suggested where "locking" essentially just locks the controller, not controller and ownership? i think we could just add another map to track locking

Reply:
Yeah that makes sense to me. Maybe also some means to see if a table is locked?